tools/matrix-diggest

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+# -*- coding:utf-8 -*-
+#
+# Author: Pablo Saavedra
+# Maintainer: Pablo Saavedra
+# Contact: saavedra.pablo at gmail.com
+
+import argparse
+import traceback
+import time
+import sys
+
+from matrixbot import utils
+from matrixbot import matrix
+
+## vars ########################################################################
+conffile = "./matrixbot.cfg"
+days = 1
+sendto = "no-reply@example.domain"
+
+## command line options parser #################################################
+parser = argparse.ArgumentParser()
+parser.add_argument("-c", "--conffile", dest="conffile", default=conffile,
+                    help="Conffile (default: %s)" % conffile)
+parser.add_argument("-s", "--sendto", dest="sendto", default=sendto,
+                    help="Sent to (default: %s)" % sendto)
```

Comment:
I assume you want "Send to (default: %s)" here.

Reply:
Fixed. Thanks for spotlighting this.